From 1639e987e8b4f0df3eba1fcad3caa0942761035c Mon Sep 17 00:00:00 2001
From: Ben Horgan <ben.horgan@arm.com>
Date: Thu, 24 Nov 2022 15:10:21 +0000
Subject: [PATCH] buildfix: Fix build for Ubuntu 22.04

Don't warn on deprecated functions as otherwise the build
fails due to the use of deprecated openssl functions and
-Werror.

Signed-off-by: Ben Horgan <ben.horgan@arm.com>
---
 test/rules.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/rules.mk b/test/rules.mk
index 53f3a83..a759b69 100644
--- a/test/rules.mk
+++ b/test/rules.mk
@@ -30,7 +30,8 @@ HOST_SRCS := \
 	$(LOCAL_DIR)/../transaction.c \
 	$(LOCAL_DIR)/block_test.c \
 
-HOST_FLAGS := -DBUILD_STORAGE_TEST=1
+HOST_FLAGS := -DBUILD_STORAGE_TEST=1 \
+	-Wno-error=deprecated-declarations
 
 HOST_LIBS := \
 	m \
-- 
2.34.1

